test/safe_math/safe_math.js

```diff
@@ -0,0 +1,122 @@
+const BigNumber = require('bn.js');
```

Comment:
Suggest renaming to "BN" to avoid future confusion since BigNumber.js is not the same as bn.js. If you disagree, please explain.